lib/utils/interfaces/auth.interface.ts

```
@@ -6,7 +10,126 @@ export interface IAuthModalProps {
+  name?: string;
+  notificationToken?: string;
+  isActive?: boolean;
+}
+
+export interface IAuthContextProps {
+  authToken: string;
+  setAuthToken: Dispatch<SetStateAction<string>>;
+  user: IUser | null;
+  setUser: Dispatch<SetStateAction<ILoginProfile | null>>;
+  checkEmailExists: (email: string) => Promise<IEmailExists>;
+  checkPhoneExists: (phone: string) => Promise<boolean | void>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)